demo/api/src/db/ormconfig.cli.ts

```diff
@@ -1,12 +1,13 @@
+import { UserContentScopes, UserPermission } from "@comet/cms-api";
 import { Options } from "@mikro-orm/core";
 import { PageTreeNodeScope } from "@src/page-tree/dto/page-tree-node-scope";
 
 import ormConfig from "./ormconfig";
 
 const config: Options = {
     ...ormConfig,
-    entities: ["./dist/**/*.entity.js", PageTreeNodeScope],
-    entitiesTs: ["./src/**/*.entity.ts", PageTreeNodeScope],
+    entities: ["./dist/**/*.entity.js", PageTreeNodeScope, UserPermission, UserContentScopes],
+    entitiesTs: ["./src/**/*.entity.ts", PageTreeNodeScope, UserPermission, UserContentScopes],
```

Comment:
I don't think you have tho add those entities here. (You can temporarily to generate migrations)

Reply:
It is necessary to create the migrations and I found this as the only way to do it (since `PageTreeNodeScope` is added also). Should it be documented somewhere to add these entities temporarily? If yes, where?